src/intro/no-std.md

```diff
@@ -1,5 +1,74 @@
 # A `no_std` Rust Environment
 
-> ❌: **This section should cover:**
->
-> * what #[no_std] means: no I/O (filesystems), allocation, etc.
\ No newline at end of file
+As mentioned earlier, this book is all about "bare metal" programming. Standard libraries in turn usually
```

Comment:
Is it? I wouldn't say so, embedded rust is meant to work a lot more systems including those with embedded OSes or even Linux running on them (like the Raspberry Pi). I think the essence is that the latter is not actually required to use the resources provided by embedded systems.